src/app/services/export.service.ts

```diff
@@ -246,23 +246,23 @@ export class ExportService {
     }
     // 2. Validate if job name is not duplicated
     for (let job of this.exportJobs) {
-      if (job['jobName'] === name) {
+      if (job.name === name) {
         const summary = 'Duplicate job name, choose a new name.';
```

Comment:
is this validation still needed? Looks like the name is validated twice now

Reply:
Yes, most of it can go now.